### PR TITLE
Adding ActiveUserPopUp

### PR DESCRIPTION
**Pull request by GitSakro**

### CODE REVIEW

File: src/main/java/textEditor/controller/EditorController.java [hunk 1]
```diff
@@ -278,6 +325,11 @@ private void fileSaveClicked() {
 
     @FXML
     private void fileCloseClicked() {
+        try {
+            activeUserHandler.removeUserToProject(project.getId(), user.getUsername());
```
Comment (niklaus64) [inline src/main/java/textEditor/controller/EditorController.java:329]: removeUserFromProject - better

File: src/main/java/textEditor/Server.java [hunk 2]
```diff
@@ -18,16 +15,20 @@ public static void main(String[] args) {
...
+            EditorModel editorModelExport = (EditorModel) UnicastRemoteObject.exportObject(editorModel, 0);
+            DatabaseModel databaseModelExport = (DatabaseModel) UnicastRemoteObject.exportObject(databaseModel, 0);
+            ActiveUserHandler activeUserHandlerExport = (ActiveUserHandler) UnicastRemoteObject.exportObject(activeUsersHandler,0);
 
             //Binding names and models interfaces
-            registry.rebind("EditorModel", editorModel);
-            registry.rebind("DatabaseModel", databaseModel);
+            registry.rebind("EditorModel", editorModelExport);
+            registry.rebind("DatabaseModel", databaseModelExport);
+            registry.rebind("ActiveUserHandler",activeUserHandlerExport);
```
Comment (mdjdrn1) [inline src/main/java/textEditor/Server.java:30]: Please add space after comma.

**Review by niklaus64 — APPROVED** [review]
Nice :+1:

File: src/main/java/textEditor/controller/PopupUsersEditingController.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+import textEditor.model.ActiveUserHandler;
+import textEditor.view.WindowSwitcher;
+
+import java.net.URL;
+import java.rmi.NotBoundException;
+import java.rmi.RemoteException;
+import java.util.ArrayList;
+import java.util.ResourceBundle;
+
+public class PopupUsersEditingController implements WindowSwitcherInjectionTarget, Initializable, ClientInjectionTarget,ProjectInjectionTarget {
```
Comment (mdjdrn1) [inline src/main/java/textEditor/controller/PopupUsersEditingController.java:17]: Reformat

File: src/main/java/textEditor/controller/ProjectController.java [hunk 4]
```diff
@@ -91,17 +99,18 @@ private void fetchUserProjectsFromDatabase() {
     private void setupProjectsListView() {
         projectListView.setItems(FXCollections.observableArrayList(this.projects));
         projectListView.getSelectionModel().getSelectedItems().addListener((ListChangeListener<? super Project>) (e) -> {
-            Project selectedProject = projectListView.getSelectionModel().getSelectedItem();
```
Comment (mdjdrn1) [inline src/main/java/textEditor/controller/ProjectController.java:94]: Why did you rename it?

File: src/main/java/textEditor/model/ActiveUsersHandlerImpl.java [hunk 5]
```diff
@@ -0,0 +1,43 @@
+package textEditor.model;
+
+import java.io.Serializable;
+import java.rmi.RemoteException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+
+public class ActiveUsersHandlerImpl implements Serializable, ActiveUserHandler {
```
Comment (mdjdrn1) [inline src/main/java/textEditor/model/ActiveUsersHandlerImpl.java:9]: Reformat

File: src/main/java/textEditor/view/WindowSwitcher.java [hunk 6]
```diff
@@ -64,10 +72,19 @@ public void loadWindow(Window window) throws IOException {
...
                 break;
             default:
+                System.err.println("Invalid window!");
                 Platform.exit();
                 System.exit(1);
         }
-        reinitializeCloseHandler();
+    }
+
+    private void loadActiveUsersPopup() throws IOException {
```
Comment (mdjdrn1) [inline src/main/java/textEditor/view/WindowSwitcher.java:81]: Use loadWindow method
